src/components/FlowRunCreateFormV2.vue

```diff
@@ -124,7 +124,7 @@
   const workQueueName = ref<string | null>(props.deployment.workQueueName)
   const retries = ref<number | null>(null)
   const retryDelay = ref<number | null>(null)
-  const jobVariables = ref<string | undefined>(can.access.flowRunInfraOverrides ? '{}' : undefined)
+  const jobVariables = ref<string | undefined>('{}')
```

Comment:
I feel like a bunch of handling could get simplified now by removing this `| undefined` in places.... 

but this should all work as-is

Reply:
it looks the API itself calls this field optional so technically the typing is _correct_ but it'd certainly make everything simpler if we stuck to a single type especially if we're defaulting to a defined value. 

probably not worth the headache atm tho tbh